internal/pkg/policy/parsed_policy.go

```diff
@@ -166,34 +148,33 @@ func parsePerms(permsRaw json.RawMessage) (RoleMapT, error) {
 	return m, nil
 }
 
-func findDefaultOutputName(outputsRaw json.RawMessage) (string, error) {
-	outputsMap, err := smap.Parse(outputsRaw)
-	if err != nil {
-		return "", err
-	}
-
+// findDefaultName returns the name of the 1st output with the "elasticsearch" type or falls back to behaviour that relies on deprecated fields.
+//
+// Previous fleet-server and elastic-agent released had a default output which was removed Sept 2021.
+func findDefaultOutputName(outputs map[string]map[string]interface{}) (string, error) {
```

Comment:
From what I can tell `ParsedPolicy.Default` is only set in the fleet-server, nothing fleet-server runs uses it (and it's not transmitted as part of a policy_change action. Do we still want to keep this?
@blakerouse I believe you wrote this, do you remember any additional context?